cmake/winml_sdk_helpers.cmake

```diff
@@ -0,0 +1,117 @@
+cmake_minimum_required(VERSION 3.0)
```

Comment:
Please add license header and copyright notice.

Reply:
done